src/App.tsx

```diff
@@ -1,27 +1,77 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
+import classNames from 'classnames';
 import { GoodsList } from './GoodsList';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+import { getAll, get5First, getRedGoods } from './api/goods';
+import { Good } from './types/Good';
+import { ActiveButton } from './types/ActiveButton';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <h1>Dynamic list of Goods</h1>
+import 'bulma/css/bulma.css';
 
-    <button type="button" data-cy="all-button">
-      Load all goods
-    </button>
+export const App: React.FC = () => {
+  const [goods, setGoods] = useState<Good[]>([]);
+  const [
+    activeButton,
+    setButtonType,
+  ] = useState<ActiveButton>(ActiveButton.NONE);
 
-    <button type="button" data-cy="first-five-button">
-      Load 5 first goods
-    </button>
+  const handleAllGoodsButton = async () => {
+    const visibleGood = await getAll();
```

Comment:
if there will be any issues on server, your code won't work
let's also add functionality for handling possible errors while making server request